server/src/resolvers/currencyAccountResolver.ts

```diff
@@ -3,25 +3,50 @@ import { CurrencyAccountsRow } from "../db/tables.js";
 import * as CurrencyAccountRepo from "../repositories/Currency/CurrencyAccountsRepository.js"
 
 export const CurrencyAccountResolvers = {
-    CurrencyAccount: {
-        owner: async (
-            parent: CurrencyAccountsRow,
-            _args: any,
-            _context: ApolloContext
-        ) => {
-            return await CurrencyAccountRepo.getAccountOwner(parent.id);
-        }
+  CurrencyAccount: {
+    owner: async (
+      parent: CurrencyAccountsRow,
+      _args: any,
+      _context: ApolloContext
+    ) => {
+      return await CurrencyAccountRepo.getAccountOwner(parent.id);
+    }
+  },
+
+  Query: {
+    currencyAccount: async (
+      _parent: any,
+      args: {
+        accountID: number,
+      },
+      { isStaff }: ApolloContext
+    ) => {
+      return isStaff(async () => (await CurrencyAccountRepo.getAccountByID(args.accountID)));
     },
 
-    Query: {
-        currencyAccount: async (
-            _parent: any,
-            args: {
-                accountID: number,
-            },
-            { isStaff }: ApolloContext
-        ) => {
-            return isStaff(async () => (await CurrencyAccountRepo.getAccountByID(args.accountID)));
-        },
+    currencyAccountsLimit: async (
```

Comment:
same comment as currencyLedgerLimit
rn by name this kinda sounds like you're getting the limit of the account (which we dont have but like thats a plaisible thing to have in your money system